pkg/server/backend_manager.go

```diff
@@ -61,7 +86,7 @@ func newBackend(conn agent.AgentService_ConnectServer) *backend {
 // connections, i.e., get, add and remove
 type BackendStorage interface {
 	// AddBackend adds a backend.
-	AddBackend(agentID string, conn agent.AgentService_ConnectServer) Backend
+	AddBackend(identifier string, idType pkgagent.IdentifierType, conn agent.AgentService_ConnectServer) Backend
 	// RemoveBackend removes a backend.
 	RemoveBackend(agentID string, conn agent.AgentService_ConnectServer)
```

Comment:
The RemoveBackend function now accepts more than just agentIDs right? Should the function signature mirror the AddBackend function?

Reply:
Change the functin signature of `RemoveBackend` to mirro the `AddBackend`